docs/content/design/historical.md

```diff
@@ -50,4 +50,11 @@ Returns the Druid version, loaded extensions, memory used, total memory and othe
 
 * `/druid/historical/v1/loadstatus`
 
-Returns a flag indicating if all segments in the local cache have been loaded. This can be used to know when a historical node is ready to be queried after a restart.
+Returns JSON of the form `{"cacheInitialized":<value>}`, where value is either `true` or `false` indicating if all
+segments in the local cache have been loaded. This can be used to know when a historical node is ready
+to be queried after a restart.
+
+* `/druid/historical/v1/readiness`
+
+Similar to `/druid/historical/v1/loadstatus`, but instead or returning JSON with a flag, responses 200 OK if segments
```

Comment:
Typo here. Should be "...instead _**of**_ returning JSON with a flag, _**returns**_ 200 OK..."

Reply:
Fixed typo. I chose "responses" because in my perception it "returns" body, which is empty in this case